pymongo/asynchronous/mongo_client.py

```diff
@@ -2354,6 +2354,13 @@ async def bulk_write(
             if not write_concern:
                 write_concern = self.write_concern
 
+        if write_concern and not write_concern.acknowledged and verbose_results:
+            raise InvalidOperation(
+                "Cannot request unacknowledged write concern and verbose results"
+            )
+        elif write_concern and not write_concern.acknowledged and ordered:
+            raise InvalidOperation("Cannot request unacknowledged write concern and ordered writes")
```

Comment:
Shouldn't there be some dead code we can remove now that we don't support these operations?